native/cocos/bindings/manual/jsb_conversions.h

```diff
@@ -1524,7 +1524,7 @@ inline bool nativevalue_to_se(const std::shared_ptr<T> &from, se::Value &to, se:
         to.setNull();
         return true;
     }
-    auto it = se::NativePtrToObjectMap::find(nativePtr);
+    auto it = se::NativePtrToObjectMap::find((void *)nativePtr);
```

Comment:
It better to add a template function in MappingUtils.h
```c++
    template<typename T>
    static std::enable_if_t<!std::is_void_v<T>, Map::iterator> find(T * v) {
        return find((void*)v);
    }
```
So that all `const T *` will be accept by NativePtrToObjectMap